images/filelogoffsetvolumeownership/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: Copyright 2025 Dash0 Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# See README.md for an explanation of this script and its purpose.
+
+# MAINTENANCE NOTE: THIS SCRIPT *DELIBERATELY* DOES NOT RUN WITH set -e! See README.md for more details.
+set -x
```

Comment:
This script and the omission of `set -e` here is the central change of this PR. IMO, under ordinary circumstances, omitting `set -e` is a non-starter, but in this specific case it might be the correct choice.

As the comment says, see the README.md next to this script an in-depth explanation.